static/gsApp/registerHooks.tsx

```diff
@@ -174,6 +175,11 @@ const GETSENTRY_HOOKS: Partial<Hooks> = {
    */
   'onboarding-wizard:skip-help': () => OnboardingWizardHelp,
 
+  /**
+   * Get list of organization roles
+   */
+  'member-invite-modal:organization-roles': getOrgRoles,
```

Comment:
uh general react Q, why is one hook do () => function vs just function obj ? Is there a difference they would both be functions ❓

Reply:
There are differences but I don't think we really care about them in this case? I am not sure
https://www.freecodecamp.org/news/the-difference-between-arrow-functions-and-normal-functions/